src/signals/incident/components/form/MapSelectors/Asset/AssetSelect.tsx

```diff
@@ -60,8 +56,11 @@ export interface AssetSelectProps {
 
 const AssetSelect: FC<AssetSelectProps> = ({ value, layer, meta, parent }) => {
   const { selection, location } = value || {}
-  const [showMap, setShowMap] = useState(false)
   const [message, setMessage] = useState<string>()
+  /**
+   * We should refactor reducers to use typescript, then use following types here instead of any.
+   */
```

Comment:
Daar ben ik het helemaal mee eens, laten we daar een story voor aanmaken en die op de backlog zetten. Ik twijfel over het nut om zo'n comment in de code te zetten.

Reply:
✅